pkg/generate/command.go

```diff
@@ -0,0 +1,35 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"github.com/budimanjojo/talhelper/pkg/config"
+	"github.com/budimanjojo/talhelper/pkg/talos"
+)
+
+func GenerateCommand(c *config.TalhelperConfig, outDir string, generateForNode string, generateApply bool, generateUpgrade bool) error {
+	if !generateApply && !generateUpgrade {
+		fmt.Printf("Must select one of `--apply` of `--upgrade`\n")
+		return nil
+	}
+
+	for _, node := range c.Nodes {
+		var isSelectedNode = generateForNode != "" && generateForNode == node.IPAddress
+		if generateForNode == "" || isSelectedNode {
+			if generateApply {
+				fileName := outDir + "/" + c.ClusterName + "-" + node.Hostname + ".yaml"
+				fmt.Printf("talosctl apply-config --talosconfig %s/talosconfig --nodes %s --file %s --insecure;\n", outDir, node.IPAddress, fileName)
```

Comment:
If `--insecure` is hardcoded, the flag name `--apply` might be confusing? Because the only usecase of `--apply-config --insecure` is for bootstrapping. Maybe change the flag name? Or maybe it's fine? I don't know really.

Reply:
Yes, I agree, I will remove the `--insecure` for now and I will try to implement some way of passing it extra flags!